src/main/java/lesson4/task8/Person.java

```diff
@@ -0,0 +1,27 @@
+package lesson4.task8;
+
+import java.util.Objects;
+
+public class Person {
+
+    private String name;
+    private String surname;
+    private String email;
+
+    public Person(String name, String surname, String email) {
+        this.name = name;
+        this.surname = surname;
+        this.email = email;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        return ((Person) obj).email.equals(this.email);
```

Comment:
so the person is equal to object of the same type when emails are equal? :)

Reply:
Maybe it would be better to add other fields to this comparison?

Reply:
Adjusted, please review.